### PR TITLE
LUCENE-8626: Lucene standardize tests part 3 and final

### PR DESCRIPTION
**Pull request by MarcusSorealheis**
# Description

I've added the final set of standardizations to test files in Lucene so that they conform to the new pattern.

# Solution

Lead with Test instead of `{something}Test.java`

# Tests

All files here are test classes. 

# Checklist

Please review the following and check all that apply:

- [x] I have reviewed the guidelines for [How to Contribute](https://wiki.apache.org/solr/HowToContribute) and my code conforms to the standards described there to the best of my ability.
- [x] I have created a Jira issue and added the issue ID to my pull request title.
- [x] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended)
- [x] I have developed this patch against the `master` branch.
- [x] I have run `./gradlew check`.
- [x] I have added tests for my changes.
- [ ] I have added documentation for the [Ref Guide](https://github.com/apache/lucene-solr/tree/master/solr/solr-ref-guide) (for Solr changes only).

### CODE REVIEW

**Comment by MarcusSorealheis** [comment]
also, for a bit of fun to show the dirty work is done: 
image

**Comment by MarcusSorealheis** [comment]
I'll fix the test failure

**Comment by erikhatcher** [comment]
lgtm!

**Comment by MarcusSorealheis** [comment]
> also, for a bit of fun to show the dirty work is done:
> image

You will notice this file returned is not actually a test class.

File: lucene/queryparser/src/test/org/apache/lucene/queryparser/surround/query/Test02Boolean.java [hunk 1]
```diff
@@ -46,11 +46,11 @@ public void setUp() throws Exception {
   SingleFieldTestDb db1;
 
   public void normalTest1(String query, int[] expdnrs) throws Exception {
-    BooleanQueryTst bqt =
-        new BooleanQueryTst(
+    TestBooleanQuery tbq =
```
Comment (msokolov) [inline lucene/queryparser/src/test/org/apache/lucene/queryparser/surround/query/Test02Boolean.java:49]: ooh fancy

**Review by msokolov — APPROVED** [review]
yeah, lgmt too, thanks!

**Pull request merged**